TriblerGUI/widgets/trustpage/style.css

```diff
@@ -7,11 +7,26 @@ html, body {
     background-color: #202020;
 }
 
+.metric {
```

Comment:
I personally think this name is somewhat unclear, shouldn't this at least be plural?